hotkey_linux.c

```diff
@@ -24,27 +43,10 @@ int displayTest() {
 	if (d == NULL) {
 		return -1;
 	}
+	XSetErrorHandler(xErrHandler);
```

Comment:
What about the default error handler? What the returned handler from XSetErrorHandler is not preserved and restored?

Reply:
I am not sure when to restore the default handler. 
Count the registered hotkeys and when all of them are unregistered restore the default handler? As we can't know when exactly the error will be raised and the key is grabbed successfully, thus we don't know when we can safely restore the default handler.
The whole X11 error handling design is so restrictive.